From 8740ed7d304a931bab958967a1eb4295c68f9f69 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Sun, 15 Sep 2024 10:32:50 +0300
Subject: [PATCH] [FSW-8694] fs1: key revoke is invalid for fw image

With this patch removed progress bit to report
key revoke. Key revoke concept related to
manifest validation and not required for firmware
image validation.

Change-Id: If273b52f47283f6de01b4d0537351aeb5c70fdc4
---

diff --git a/common/status.h b/common/status.h
index 4393d7f..3f06c23 100644
--- a/common/status.h
+++ b/common/status.h
@@ -24,9 +24,8 @@
 	PROGRESS_BRINGUP_UNALIGNED_IMG_SIZE,
 	PROGRESS_BRINGUP_DMA_ERROR,
 	PROGRESS_BRINGUP_DMA,
-	PROGRESS_BRINGUP_KEY_REVOKED,
 	PROGRESS_BRINGUP_KEY_AUTH_FAIL,
-	PROGRESS_BRINGUP_SECURITY,
+	PROGRESS_BRINGUP_VERIFY_SUCCESS,
 };
 
 enum boot_rom_boot_desc_progress {
